docs/components/concepts/backups.md

```diff
@@ -8,9 +8,9 @@ description: "Learn more about Backups in Camunda 8 SaaS."
 
 You can use the backup feature of Camunda 8 SaaS to regularly back up the state of all of its components (Zeebe, Operate, Tasklist, and Optimize) with _zero downtime_. In case of failures that lead to data loss, you can request to restore the backup.
 
-A Camunda 8 SaaS backup consists of a data backup of Zeebe, Operate, Tasklist, Optimize, and the backup of exported Zeebe records in Elasticsearch. Since the data of these applications depend on each other, the backup must be consistent across all components. Therefore, the backup of a Camunda 8 Cluster is taken as a whole.
+A Camunda 8 SaaS backup consists of a data backup of Zeebe, Operate, Tasklist, Optimize, and the backup of exported Zeebe records in Elasticsearch. Since the data of these applications depend on each other, the backup must be consistent across all components. Therefore, the backup of a Camunda 8 cluster is taken as a whole.
```

Comment:
"the backup of a Camunda 8 cluster is taken as a whole" -- what do we mean by this?

Reply:
Instead of individual backups per component, the scope of the backup is the full cluster-worth of components. You cannot take an individual backup of just Operate or just Tasklist. @multani should be able to confirm this as part of SRE.